### PR TITLE
Categories table

### LINKED ISSUE #897: Categories Report: Table
Child task of #170.

This task involves creating the table from API data for the Categories Report.

### PR DESCRIPTION
**Pull request by Aljullu**
Fixes #897.

Adds a Table to the _Categories_ report.

Please, notice the data is loaded from SwaggerHub, for now.

### Screenshots
image

### Detailed test instructions:
- Go to the _Categories_ report.
- Verify there is a table and interact with it.

### CODE REVIEW

File: client/analytics/report/categories/table.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+	}
+
+	getSummary( totals ) {
+		if ( ! totals ) {
+			return [];
+		}
+		return [
+			{
+				label: _n( 'category', 'categories', totals.categories_count, 'wc-admin' ),
+				value: totals.categories_count,
```
Comment (timmyc) [inline client/analytics/report/categories/table.js]: `numberFormat()` again on all of these non-currency items would be good 👍

File: client/analytics/report/categories/table.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+		const headers = this.getHeadersContent();
+		const orderedCategories = orderBy( items, query.orderby, query.order );
+		const rows = this.getRowsContent( orderedCategories );
+		const totalRows = get( primaryData, [ 'data', 'totals', 'categories_count' ], items.length );
+		const summary = primaryData.data.totals ? this.getSummary( primaryData.data.totals ) : null;
+
+		return (
+			<TableCard
+				title={ __( 'Coupons', 'wc-admin' ) }
```
Comment (timmyc) [inline client/analytics/report/categories/table.js]: Should be Categories here and on the next line correct?

**Review by timmyc — COMMENTED** [review]
Couple of items to update, otherwise looking good and testing out well.

File: client/analytics/report/categories/table.js (hunk 1 above)
Comment (Aljullu) [reply client/analytics/report/categories/table.js]: Sure!

File: client/analytics/report/categories/table.js (hunk 2 above)
Comment (Aljullu) [reply client/analytics/report/categories/table.js]: Right. :sweat_smile:

**Comment by Aljullu** [comment]
@timmyc thanks for the review! I fixed the issues you noted in the comments.

**Comment by psealock** [comment]
I can give this a spin once #928 gets merged and this branch updated.

**Comment by Aljullu** [comment]
@psealock I updated this branch with the last version of `master` after #928 was merged. Should be ready for review again.

**Comment by timmyc** [comment]
Testing out well for me still, getting this one merged in prior to the release.

**Pull request merged**